code/datums/ruins/lavaland.dm

```diff
@@ -13,6 +13,8 @@
 	description = "Seemingly plucked from a tropical destination, this beach is calm and cool, with the salty waves roaring softly in the background. \
 	Comes with a rustic wooden bar and suicidal bartender."
 	suffix = "lavaland_biodome_beach.dmm"
+	cost = 0
+	always_place = TRUE
```

Comment:
Beach biodome has been removed from the code I believe. Update this when you can please

Reply:
Oof, yes. Please merge master if you haven't recently; good catch contra

Reply:
> Beach biodome has been removed from the code I believe. Update this when you can please

it's TMed back in rn, also yeah, the conflicted pr will need a merge master